formats/placeholder.jsonnet

```diff
@@ -53,13 +70,423 @@
+              },
+            },
+          },
+          description: "Used for organisations, to allow us to publish featured links. Remove when organisations are migrated.",
+        },
+        ordered_featured_documents: {
+          type: "array",
+          items: {
+            type: "object",
+            additionalProperties: false,
+            required: [
+              "title",
+              "href",
+              "image_href",
+              "summary",
+              "published_at",
+              "document_type",
+            ],
+            properties: {
+              title: {
+                type: "string",
+              },
+              href: {
+                type: "string",
+                format: "uri",
+              },
+              image_href: {
+                type: "string",
+                format: "uri",
+              },
+              summary: {
+                type: "string",
+              },
+              published_at: {
+                type: "string",
+              },
+              document_type: {
+                type: "string",
+              },
+            },
+          },
+          description: "Used for organisations, to allow us to publish featured documents. Remove when organisations are migrated.",
+        },
+        ordered_ministers: {
+          type: "array",
+          items: {
+            type: "object",
+            additionalProperties: false,
+            required: [
+              "name",
+              "role",
+              "href",
+            ],
+            properties: {
+              name: {
+                type: "string",
+              },
+              role: {
+                type: "string",
+              },
+              href: {
+                type: "string",
+                format: "uri",
+              },
+              image_href: {
+                type: "string",
+                format: "uri",
+              },
+            },
+          },
+          description: "Used for organisations, to allow us to publish minister information. Remove when organisations are migrated.",
+        },
+        ordered_board_members: {
+          type: "array",
+          items: {
+            type: "object",
+            additionalProperties: false,
+            required: [
+              "name",
+              "role",
+              "href",
+            ],
+            properties: {
+              name: {
+                type: "string",
+              },
+              role: {
+                type: "string",
+              },
+              href: {
+                type: "string",
+                format: "uri",
+              },
+              image_href: {
+                type: "string",
+                format: "uri",
+              },
+            },
+          },
+          description: "Used for organisations, to allow us to publish board member information. Remove when organisations are migrated.",
+        },
+        ordered_military_personnel: {
```

Comment:
Do we have intentions of extracting this into another schema in the future?

Reply:
I think once we migrate people and roles then we'll make these links in the links hash so they get updated automatically using dependency resolution.